### DISCUSSION TITLE
body_y_offset is not working

### DISCUSSION THREAD

**Discussion by eDeNZeSk** [SunOner/yolov8_aimbot#175] [Q&A]
When i change body_y_offset settings at config.ini is no change anything. How AI will aim at head or upper body?

**Comment by SunOner** [comment]
Try to change disable_headshot option.

**Comment by Bazz0r** [comment]
I can confirm this issue. When disable_headshot is yes and I enable the overly with boxes, the debug window is showing a bounding box around the head and body and the aim is moving to the head. This did not happen in a previous release, thanks.

**Reply by eDeNZeSk** [reply]
How i will fix it?

**Reply by Bazz0r** [reply]
This isn't answered, I was confirming I had the same problem.
